docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -53,6 +53,25 @@ The component doesn't base64-encode the value provided for `auth`.
 Please make sure that you store the `auth` value as base64 in Vault.
 ====
 
+== `caBundle`
+
+[horizontal]
+type:: string
+default:: `null`
+example::
++
+[source,yaml]
+----
+caBundle: | <1>
+  -----BEGIN CERTIFICATE-----
+  MY AWESOME CA
+  -----END CERTIFICATE-----
+----
+<1> Custom CA deployed as ConfigMap `custom-ca-bundle`.
+
+This parameter allows to deploy a ConfigMap in the `openshift-config` namespace.
+The ConfigMap contains the configured CA in the key `ca-bundle.crt`, to be used by other components.
```

Comment:
```suggestion
The ConfigMap contains the configured CA in the key `ca-bundle.crt`, to be used by other components.

[IMPORTANT]
====
This ConfigMap isn't used by OpenShift components.
OpenShift uses a separate config map called `user-ca-bundle` in namespace `openshift-config` to extend the system wide trusted CA bundle.
See parameter `trustedCA` in component https://hub.syn.tools/openshift4-proxy/references/parameters.html#_trustedca[openshift4-proxy] for details on how to deploy additional trusted CA certificates for the system-wide trusted CA bundle.
====
```

Reply:
depends... the most common use case for extending the system CA bundle is to deploy an internal CA certificate that's used to issue the wildcard certificate for an outgoing HTTP proxy, so it makes some sense to have the `trustedCA` config in `openshift4-proxy`.

Also off the top of my head I'm not sure whether updating the `user-ca-bundle` config map also triggers a machineconfig update (which may or may not drain & reboot nodes, e.g. to update the crio/containerd config to use the additional trusted CA certificates).